lib/active_fedora/file.rb

```diff
@@ -124,8 +131,13 @@ def content_changed?
       local_or_remote_content(false) != @ds_content
```

Comment:
Yes, this bare `elsif` is from the original PR from @tpendragon which @cjcolvar squashed and rebased which @jcoyne  reverted which I attempted to unrevert. That's a lot of developers passing up the tantalizing opportunity to nuke a bare `elsif` but I agree that it should go.